src/Interpreters/Streaming/tests/gtest_streaming_aggregator.cpp

```diff
@@ -0,0 +1,389 @@
+#include <cstddef>
+#include <cstdlib>
+#include <memory>
+#include <utility>
+#include <vector>
+#include <AggregateFunctions/AggregateFunctionFactory.h>
+#include <Columns/ColumnLowCardinality.h>
+#include <Columns/IColumn.h>
+#include <Core/Block.h>
+#include <DataTypes/DataTypeFactory.h>
+#include <DataTypes/DataTypeLowCardinality.h>
+#include <Formats/formatBlock.h>
+#include <IO/ReadBufferFromString.h>
+#include <IO/WriteBufferFromString.h>
+#include <Interpreters/CollectJoinOnKeysVisitor.h>
+#include <Interpreters/Context.h>
+#include <Interpreters/JoinedTables.h>
+#include <Interpreters/Streaming/Aggregator.h>
+#include <Interpreters/Streaming/HashJoin.h>
+#include <Interpreters/Streaming/SyntaxAnalyzeUtils.h>
+#include <Interpreters/Streaming/TrackingUpdatesData.h>
+#include <Interpreters/Streaming/WindowCommon.h>
+#include <Interpreters/TranslateQualifiedNamesVisitor.h>
+#include <Parsers/ASTIdentifier.h>
+#include <Parsers/ASTLiteral.h>
+#include <Parsers/ASTSelectQuery.h>
+#include <Parsers/ExpressionElementParsers.h>
+#include <Parsers/ParserSelectQuery.h>
+#include <Parsers/parseQuery.h>
+#include <Parsers/queryToString.h>
+#include <Processors/Formats/IRowInputFormat.h>
+#include <Processors/Transforms/Streaming/JoinTransform.h>
+#include <base/constexpr_helpers.h>
+#include <gtest/gtest.h>
+#include <Common/logger_useful.h>
+#include <Common/tests/gtest_global_context.h>
+#include <Common/tests/gtest_global_register.h>
+#include "Columns/ColumnString.h"
+#include "Columns/ColumnsNumber.h"
+#include "Core/ColumnNumbers.h"
+#include "Core/NamesAndTypes.h"
+#include "Core/iostream_debug_helpers.h"
+#include "DataTypes/DataTypeDateTime.h"
+#include "DataTypes/DataTypeDateTime64.h"
+#include "DataTypes/DataTypeFixedString.h"
+#include "DataTypes/DataTypeInterval.h"
+#include "DataTypes/DataTypeString.h"
+#include "DataTypes/DataTypesNumber.h"
+#include "DataTypes/IDataType.h"
+#include "Interpreters/Streaming/TableFunctionDescription.h"
+#include "Interpreters/Streaming/TableFunctionDescription_fwd.h"
+
+namespace DB
+{
+using namespace DB;
+
+void prepareHeader(Block & header)
+{
+    std::vector<ColumnWithTypeAndName> columns;
+    columns.push_back(ColumnWithTypeAndName(DataTypeFactory::instance().get("int8"), fmt::format("int8", 0)));
```

Comment:
I don't understand this parameter `fmt::format("int8", 0)`, just expected a column name here